core/object/script_language_extension.h

```diff
@@ -205,7 +205,7 @@ class ScriptExtension : public Script {
 
 	GDVIRTUAL0RC(Variant, _get_rpc_config)
 
-	virtual const Variant get_rpc_config() const override {
+	virtual Variant get_rpc_config() const override {
```

Comment:
This may be an unavoidable limitation but the fact that the return type is Variant and not Dictionary is quite odd. Not sure we should expose methods this way, *if it can be avoided*